pkg/gatherers/conditional/gather_logs_of_unhealthy_pods.go

```diff
@@ -0,0 +1,108 @@
+package conditional
+
+import (
+	"context"
+	"fmt"
+
+	"k8s.io/client-go/kubernetes"
+	v1 "k8s.io/client-go/kubernetes/typed/core/v1"
+	"k8s.io/klog/v2"
+
+	"github.com/openshift/insights-operator/pkg/gatherers"
+	"github.com/openshift/insights-operator/pkg/gatherers/common"
+	"github.com/openshift/insights-operator/pkg/record"
+)
+
+// BuildGatherLogsOfUnhealthyPods collects either current or previous logs for pods firing one of the configured alerts.
+//
+// * Location in archive: conditional/unhealthy_logs/<namespace>/<pod>/<container>/[current|previous].log
```

Comment:
so far, in conditional gatherer we use format where we put the type of the item first and then the name (for example `conditional/namespaces/{namespace}/pods/{pod_name}/containers/{container_name}/logs/last-{n}-lines.log`), I'd suggest to follow this concept

Reply:
ok but what about previous/current log? Would it be `conditional/namespaces/{namespace}/pods/{pod_name}/containers/{container_name}/logs/previous.log` or `conditional/namespaces/{namespace}/pods/{pod_name}/containers/{container_name}/logs/previous-last-{n}-lines.log`?

Reply:
I'd do `conditional/namespaces/{namespace}/pods/{pod_name}/containers/{container_name}/previous-logs/last-{n}-lines.log`, but `previous-last-{n}-lines.log` would also work

Reply:
ok both works for me as well. So the suggestion is to update it to use one of these suggested paths and then deal with possible duplicates in a new spike right? I think I am fine with this approach. @natiiix WDYT?